sdk/eventhub/azure-eventhubs/azure/eventhub/aio/_consumer_client_async.py

```diff
@@ -0,0 +1,187 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import logging
+from typing import Any, Union, TYPE_CHECKING, Callable, Dict, List, Tuple
+from azure.eventhub import EventPosition, EventData, \
+    EventHubSharedKeyCredential, EventHubSASTokenCredential
+from .eventprocessor.event_processor import EventProcessor, CloseReason
+from .eventprocessor.partition_context import PartitionContext
+from .client_async import EventHubClient
+if TYPE_CHECKING:
+    from azure.core.credentials import TokenCredential  # type: ignore
+
+log = logging.getLogger(__name__)
```

Comment:
logger name convention is `_LOGGER`.

Reply:
Yes, we need to change this var name in all files later. That impacted too many files so we better use a different PR for files later.

Reply:
Not blocking for preview5, solving